packages/techdocs-common/src/stages/publish/azureBlobStorage.ts

```diff
@@ -350,4 +380,30 @@ export class AzureBlobStoragePublish implements PublisherBase {
 
     await Promise.all(promises);
   }
+
+  protected async getAllBlobsFromContainer({
+    prefix,
+    maxPageSize,
+  }: {
+    prefix: string;
+    maxPageSize: number;
+  }): Promise<string[]> {
+    const blobs: string[] = [];
+    const container = this.storageClient.getContainerClient(this.containerName);
+
+    let iterator = container.listBlobsFlat({ prefix }).byPage({ maxPageSize });
+    let response = (await iterator.next()).value;
+
+    do {
+      for (const blob of response.segment.blobItems) {
```

Comment:
How will the code behave on this line if the bucket is empty?
Maybe we can test to make sure `segment.blobItems` won't be undefined...